source/scss/components/_rating.scss

```diff
@@ -0,0 +1,24 @@
+.rating {
+  border-radius: 2px;
+  color: $white;
+  font-weight: $font-weight-bold;
+  padding: 1px 0;
+  text-align: center;
+  width: 50px;
+
```

Comment:
In the invision file is a height defined, but here not? Is it not needed or does the height is defined at another place?

Reply:
Yes, the height is defined by the padding. The reason is, just in case, a user is changing a font-size or even the font itself, the content would not overflow the content.

It's the same with the width. I made the designer and stakeholder aware of that, that is can happen. But they decided to take the width definition, instead of using a padding. Here a width makes sense also, because anywhere, where it would be implemented, it would make sense to have an explicit width.

The computed height of the element is 28px, as requested.

Reply:
yes I agreed